source/_integrations/alarm_control_panel.markdown

```diff
@@ -25,16 +25,18 @@ Please see [manual alarm](/integrations/manual) or [template alarm](/integration
 
 An alarm panel entity can have the following states. Not all integrations implement all the different states.
 
-- **Disarmed**: The alarm is disarmed (off).
-- **Armed home**: The alarm is armed in home mode.
-- **Armed away**: The alarm is armed in away mode.
-- **Armed night**: The alarm is armed in night mode.
-- **Armed vacation**: The alarm is armed in vacation mode.
-- **Armed custom bypass**: The alarm is armed in bypass mode.
-- **Pending**: The alarm is pending (towards triggered).
-- **Arming**: The alarm is in the process of being armed.
-- **Disarming**: The alarm is in the process of being disarmed.
-- **Triggered**: The alarm has been triggered.
+- **disarmed**: The alarm is disarmed (off).
+- **armed home**: The alarm is armed in home mode.
```

Comment:
Should we write the exact string that is set, ie `"armed_home"` or is it enough to use the same name?